### ISSUE TITLE
Synthèse : champs last_action non mis à jour

### ISSUE THREAD

**Issue by bouttier** [PnX-SI/GeoNature#2386]
Le champs ``last_action`` doit indiquer la dernière action ayant concerné une ligne : I(nsert) ou U(pdate). Cependant, ce champs possède plusieurs problème :
- pas de valeur par défaut, ``NULL`` autorisé
- non mis à jour lors d’une modification

Il est correctement complété uniquement pour les données en provenant d’OccTax, dont les triggers d’insertion et de mise à jour remplisse correctement le champs ``last_action``.

Pour pouvoir utiliser ce champs de manière plus globale que les seules données en provenance d’OccTax, il faudrait probablement :
- Définir une valeur par défaut ``I'`
- Rajouter une contrainte ``NOT NULL``
- Modifier le trigger ``gn_synthese.tri_meta_dates_change_synthese`` afin d’également mettre à jour le champs ``last_action`` à ``U``
- Modifier les différents trigger d’OccTax pour ne plus toucher au champs ``last_action``

**Comment by DonovanMaillard** [comment]
particulièrement utile en effet dans le cadre d'un usage de gn2pg par exemple :)

**Comment by camillemonchicourt** [comment]
Oui, c'est pour ça que dans la PR qui ajoute la route d'historisation de la Synthèse pour GN2PG, en cours d'intégration (https://github.com/PnX-SI/GeoNature/pull/2337), @bouttier a fait des modifications pour ne pas utiliser la colonne ``last_action`` pour le moment, mais se baser sur ``meta_create_date`` et  ``meta_update_date`` : 
- Si ``meta_create_date`` = ``meta_update_date`` : la dernière action est une création
- Si ``meta_create_date`` < ``meta_update_date`` : la dernière action est une modification
